azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/utils/WebContainerUtils.java

```diff
@@ -121,4 +121,10 @@ public static WebContainer parseWebContainer(String webContainer, JavaVersion ja
         Collections.sort(result);
         return result;
     }
+
+    public static com.microsoft.azure.toolkits.appservice.model.WebContainer toLibraryWebContainer(String input) {
```

Comment:
when not move the condition check into `WebContainer` itself and use `WebContainer.fromString`(rename it?).
otherwise, I doubt if the implementation is complete and safe.

Reply:
for the upper method `getAvailableWebContainer`, shouldn't return a list of `WebContainer`? why it returns a list of string???

Reply:
@wangmingliang-ms maven plugin accepts input like `JAVA SE`, `JAVA 8` and will convert them to correspond web container, which differs from the constant value, so add the convert here

Reply:
the upper method `getAvailableWebContainer` is used by `ConfigMojo`, will be refactored in later pr